ci_build_images/msan.Dockerfile

```diff
@@ -1,56 +1,75 @@
 # msan.Dockerfile
 # this is to create images with MSAN for BB workers
-ARG CLANG_VERSION=15
+ARG CLANG_VERSION=19
```

Comment:
This should be 20, right? Or do we intentionally want to retain some support for clang-19? Later in the script we’re comparing `CLANG_VERSION` to even smaller version numbers.

Reply:
There's is bit in the .github/workflows that overrides this to 20. Yes, could be changed to 20 here, I was just making sure there wasn't an aversion to using it.

I was aiming on preserving compat with pre-19, but I think that's dropped in an attempt to get 19/20 working and bits could be cleaned out. There's a small amount of compat with the bleeding edge 21.